src/conn/README.md

```diff
@@ -0,0 +1,32 @@
+# Conn
+
+## Compaction
+
+### Overview
+
+Compaction is implemented in [conn_compact.c](conn_compact.c).
+
+### When will a compaction action be skipped
+
+The code implementation is [__background_compact_should_skip](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/src/conn/conn_compact.c#L233-L311).
+
+The key skip judges are (order matters):
+1. A table can be configured to skip compaction using `exclude=["table:a.wt"]`, the configuration is parsed in [__background_compact_exclude_list_process](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/src/conn/conn_compact.c#L100-L120) , you can find an example in [test_compact06](https://github.com/wiredtiger/wiredtiger/blob/cec2d68c2a51be610745b4ab481c572aaec08d3c/test/suite/test_compact06.py#L38)
+1. File size < `1MB`
+1. `background_compact.max_file_skip_time` (unit of Seconds), If the elapsed time since the last compaction run on the same URI exceeds this threshold and the device is ready, the compaction.
```

Comment:
What do you mean by device?
```suggestion
1. `background_compact.max_file_skip_time` (unit of Seconds). If the elapsed time since the last compaction run on the same URI exceeds this threshold and the device is ready, the compaction.
```